pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -0,0 +1,3165 @@
+        """
+        self.write("STORETHRU")
+
+    sweep_couple = Instrument.control(
+        "SWPCPL?", "SWPCPL %s",
+        """
+        Select either a stimulus-response or spectrum-analyzer
+        auto-coupled sweep time. In stimulus-response mode, auto-coupled sweep times are usually
+        much faster for swept-response measurements. Stimulus-response auto-coupled sweep times
+        are typicallly valid in stimulus-response measurements when the system’s frequency span is
+        less than 20 times the bandwidth of the device under test.
+
+        Type: :code:`str` or :class:`SweepCoupleMode`
+        """,
+        validator=strict_discrete_set,
+        values=[e for e in SweepCoupleMode]
+    )
+
+    sweep_output = Instrument.control(
+        "SWPOUT?", "SWPOUT %s",
+        """
+        Select the sweep-related signal that is available from J8 on the rear
+        panel. FAV provides a dc ramp of 0.5V/GHz. RAMP provides a 0—10 V ramp corresponding
+        to the sweep ramp that tunes the first local oscillator (LO). For the HP 8561B, in multiband
+        sweeps one ramp is provided for each frequency band.
+
+        Type: :code:`str` or :class:`SweepOut`
+        """,
+        validator=strict_discrete_set,
+        values=[e for e in SweepOut]
+    )
+
+    trace_data_format = Instrument.control(
+        "TDF?", "TDF %s",
+        """
+        Select the format used to input and output trace data (see the
+        TRA/TRB command, You must specify the desired format when
+        transferring data from the spectrum analyzer to a computer; this is optional when
+        transferring data to the analyzer.
+
+        Type: :code:`str` or :class:`TraceDataFormat`
+
+        .. warning::
+            Only needed for manual read out of trace data. Don't use this if you don't know what
+            You are doing.
+        """,
+        validator=strict_discrete_set,
+        values=[e for e in TraceDataFormat]
+    )
+
+    threshold = Instrument.control(
+        "TH?", "TH %s",
+        """
+        Control the minimum amplitude level and clips data at this value. Default
+        value is -90 dBm. See also - :attr:`marker_threshold` does not clip data below its threshold
+
+        Type: :code:`str, float` range -200 to 30
+
+        .. note::
+            When a trace is in max-hold mode, if the threshold is raised above any of the
+            trace data, the data below the threshold will be permanently lost.
+        """,
+        validator=joined_validators(strict_discrete_set, strict_range),
+        values=[["ON", "OFF"], arange(-200, 30)],
+        set_process=lambda v: v if isinstance(v, str) else ("%.2E" % v)
+    )
+
+    def title(self, string):
+        """Sets character data in the title area of the display, which is in
+        the upper-right corner.
+
+        A title can be up to two rows of sixteen characters each, Carriage
+        return and line feed characters are not recommended.
+        manual.
```

Comment:
The documentation here ends with the word "manual.", this could be worked into a full reference tothe manual, e.g. " See Chapter 3.14 (page 27.1) for more details"